### DISCUSSION TITLE
How do I transfer a repository, I created, to an organisation, I am 'owner' of?

### DISCUSSION THREAD

**Discussion by Papa-26** [community/community#50424] [Actions] [answered]
### Select Topic Area

Question

### Body

One year ago I had started an organisation around a topic area. Until recently it had one project repository. About three months ago I started building a new project involving the same topic area. I wasn't savvy enough to start it under the organisation's umbrella. Now I would like to transfer the mature project to the organisation, but I haven't found the appropriate 'secret formula'. On an organisation page I can only add a virginal new project, and the search function can't find my existing project from the organisational page.
How do I do it?

**Comment by neutrinomuon** [comment, marked as answer]
Hello,

If you access the repository settings and scroll down, you'll come across the '**Danger Zone**' section which includes the option to transfer ownership of the repository. This allows you to transfer the repository to another user or organization.

I hope this information is helpful.

**Reply by shreyanshshandilya15** [reply]
I got a confirmation mail on the email account linked to transfered account , but after clicking on it to accept , github page opens up and prints on screen "Oops! That repository transfer request is invalid. Did you copy the URL correctly?" . i don't know why am i getting it but it is an official mail from github. Can u please help me with this !!!

**Comment by Papa-26** [comment]
Thanks neutrinomuon!
that did it perfectly. I never would have found it without your help.

**Reply by neutrinomuon** [reply]
Glad to hear that you were able to locate the solution you were looking for! If you don't mind, could you please mark this thread as "answered"? Doing so will not only help us keep track of resolved issues, but it will also benefit other community members who may have a similar question in the future. Thank you!

**Reply by marcosares** [reply]
Hello, in my case this is not working. I get back the following error: "you don't have the permission to create public repositories on aequaland". How can I change the setting of my organization named "aequaland" for which I am owner of so that it accepts the external transfer?

**Reply by marcosares** [reply]
My bad, I was no longer an owner. Once I asked to regain access as owner or member in the targetted organization, it worked.

**Reply by neutrinomuon** [reply]
Hi, @marcosares, Please let me know if it works :)! Good luck

**Reply by marcosares** [reply]
Yes, it did work. The bottom line is that in order for the transfer to work, not only the transfer needs to be asked from the source repository via the Danger zone section but you also need to make sure that you whoever is initiating the transfer is either owner or member on the target organization.

**Reply by neutrinomuon** [reply]
Great :)! Thanks for your input

**Comment by Hanimi-Golamari-22202328** [comment]
Hi there,so when I accept the link from auto-generated mail for my repo transfer,it says something went wrong,I want to move one of my repo from my school account to personal account,so let me if there is anything for me ,Thank you 
![image](https://github.com/community/community/assets/135073203/691dd63e-11cb-4823-a396-1842678b5ba3)

**Reply by Mariel0578** [reply]
Please help me guide me

**Comment by snugly-pixel** [comment]
I am getting an error saying below the. I initiate transfer

org-**** has insufficient collaborator seats.

What does this mean? I am the owner of the repository on my personal account and owner of the organisation as well.
What needs to be done here? Any help appreciated

**Reply by ppreshant** [reply]
You must remove collaborators before moving and add them back later says [stackoverflow](https://stackoverflow.com/a/58786953/9049673)

**Comment by attardi** [comment]
In the current web interface, there is no '**Danger Zone**' in **Settings**.

**Comment by Mehrab-Javadzadeh** [comment]
Hi! 👋

You don't need to create a new repository. GitHub allows you to transfer an existing repository to an organization while keeping its history and settings.

To transfer your repository:

1. Open the repository you want to move.
2. Go to **Settings**.
3. Scroll down to the **Danger Zone** section.
4. Click **Transfer ownership**.
5. Enter the organization name and confirm the transfer.

Before transferring, make sure that:
- You have **admin access** to the repository.
- You have permission to create repositories in the organization.
- The repository name is available in the organization.

After the transfer, your repository's issues, pull requests, stars, forks, releases, and commit history will remain unchanged.

If you don't see the transfer option, check your permissions or ask an organization owner to help with the transfer.

Hope this helps! 🚀
